sdk/tests/conformance2/rendering/draw-buffers.html

```diff
@@ -375,31 +375,37 @@
 
   checkAttachmentsForColor(attachments, [0, 255, 0, 255]);
 
-  // GLES3 spec section 3.9.2 Shader Outputs
-  debug("test that gl_FragColor only writes to color number zero");
+  // By default, gl_FragColor only writes to color number zero.
+  // drawBuffers can specify the color buffer that gl_FragColor should write into.
+  // See GLES3 spec section 4.2.1 Selecting Buffers for Writing.
+  debug("test that gl_FragColor writes to color buffer specified by drawBuffers");
   gl.bindFramebuffer(gl.FRAMEBUFFER, fb);
-  gl.drawBuffers(bufs);
+  var bufNum = 4;
+  var blueBufs = makeColorAttachmentArray(bufNum);
+  gl.drawBuffers(blueBufs);
   gl.useProgram(blueProgramESSL1);
```

Comment:
This is actually already specified: https://www.khronos.org/registry/webgl/specs/latest/2.0/#5.13